package.json

```diff
@@ -69,6 +70,7 @@
     "lodash": "^4.17.20",
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
+    "react-input-slider": "^6.0.0",
```

Comment:
We have a slider component in `@flybywiresim/react-components`, can you try using that ?

Reply:
I am, this was an old component I used during my initial tests, i'll remove it from the package